kafka-init/src/main/java/io/strimzi/kafka/init/Main.java

```diff
@@ -15,10 +18,14 @@ public class Main {
 
     public static void main(String[] args) {
 
-        LOGGER.info("Init-kafka {} is starting", Main.class.getPackage().getImplementationVersion());
+        final String strimziVersion = Main.class.getPackage().getImplementationVersion();
+        LOGGER.info("Init-kafka {} is starting", strimziVersion);
         InitWriterConfig config = InitWriterConfig.fromMap(System.getenv());
 
-        KubernetesClient client = new KubernetesClientBuilder().build();
+        final String userAgent = "fabric8-kubernetes-client/" + Version.clientVersion() 
+                                + " strimzi-init-kafka/" + strimziVersion;
+        final Config kubernetesClientConfig = new ConfigBuilder().withUserAgent(userAgent).build();
+        KubernetesClient client = new KubernetesClientBuilder().withConfig(kubernetesClientConfig).build();
```

Comment:
Would it be better to avoid creating the variables in `main` which are only used once or twice at startup?